src/core/sys/posix/netinet/in_.d

```diff
@@ -972,23 +972,4 @@ else version( Android )
 IPPROTO_RAW
 */
 
-version( linux )
-{
-    enum uint IPPROTO_RAW = 255;
-}
-else version( OSX )
-{
-    enum uint IPPROTO_RAW = 255;
-}
-else version( FreeBSD )
-{
-    enum uint IPPROTO_RAW = 255;
-}
-else version( Solaris )
-{
-    enum uint IPPROTO_RAW = 255;
-}
-else version( Android )
-{
-    enum uint IPPROTO_RAW = 255;
-}
+enum uint IPPROTO_RAW = 255;
```

Comment:
No reason that I know of.  Is there any reason why [`signal` has to be provided by the local libc on every platform](https://github.com/D-Programming-Language/druntime/blob/master/src/core/stdc/signal.d#L76)?  Because [it isn't in the libc in bionic, it's in the headers (lines 117-126)](https://android.googlesource.com/platform/bionic/+/kitkat-release/libc/include/signal.h), which means the linked druntime definition doesn't work and will need an alias to bsd_signal for bionic.  I just noticed this yesterday, when trying to link a small app using `signal`, which is why the alias isn't there yet.

This could be true for practically every single declaration in druntime, ie they may be different than we expect.  Are you suggesting that we put _every_ single declaration in druntime in such a version(OS) block, just in case we need to change it later?  That would make druntime 10 times bigger.

My point has always been that if `IPPROTO_RAW` is the same on all _currently_ supported platforms, clearly there's no need to break it out like this right now.  If we eventually support a platform where it's different, let's break out the version(OS) block then.  If we _already_ know it's different for a platform that we'd like to support, fine, leave this version block in along with a comment why you're doing it.  But nobody has been able provide such concrete reasons for all these blocks.  If they did, I'd gladly put the comment in and leave that version block alone.